tests/cron/test_jobs.py

```
@@ -245,3 +247,39 @@ def todo_test_simple_email_job():
+    machine_id = 1
+
+    job_id = Job.insert('run', user_id=1, name=name, url=url, branch=branch, filename=filename, machine_id=machine_id)
+
+    # Store credentials encrypted in the DB — this is what the API endpoint does
+    User(1).update_docker_credentials([{
+        'registry': 'https://index.docker.io/v1/',
+        'username': os.getenv('GMT_TESTING_DOCKER_USER'),
+        'password': os.getenv('GMT_TESTING_DOCKER_PAT'),
+    }])
+
+    ps = subprocess.run(
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)